package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@csstools/postcss-preset-env-mono-repo",
```

Comment:
probably needs to update the name to be postcss-plugins

Reply:
Indeed :)